das/local_disk_das.go

```diff
@@ -8,34 +8,43 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/offchainlabs/nitro/arbutil"
-	"github.com/offchainlabs/nitro/solgen/go/bridgegen"
 	"os"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+
 	"github.com/offchainlabs/nitro/arbstate"
+	"github.com/offchainlabs/nitro/arbutil"
 	"github.com/offchainlabs/nitro/blsSignatures"
+	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"github.com/offchainlabs/nitro/solgen/go/bridgegen"
+
 	flag "github.com/spf13/pflag"
 )
 
 var ErrDasKeysetNotFound = errors.New("no such keyset")
 
 type LocalDiskDASConfig struct {
-	KeyDir                string `koanf:"key-dir"`
-	PrivKey               string `koanf:"priv-key"`
-	DataDir               string `koanf:"data-dir"`
-	AllowGenerateKeys     bool   `koanf:"allow-generate-keys"`
-	L1NodeURL             string `koanf:"l1-node-url"`
-	SequencerInboxAddress string `koanf:"sequencer-inbox-address"`
-	StorageType           string `koanf:"storage-type"`
+	KeyDir                string               `koanf:"key-dir"`
+	PrivKey               string               `koanf:"priv-key"`
+	DataDir               string               `koanf:"data-dir"`
```

Comment:
Since we have `--storage.s3.*`, `--storage.redis.*`, `--storage.big-cache.*`, what do you think about moving this to `LocalConfig` object so we have `--storage.local.data-dir`?  Give it a bit more symmetry.

Reply:
On the theme of symmetry, what do you think about mirroring the configuration in Nitro?

```type PersistentConfig struct {
	GlobalConfig string `koanf:"global-config"`
	Chain        string `koanf:"chain"`
}

var PersistentConfigDefault = PersistentConfig{
	GlobalConfig: ".arbitrum",
	Chain:        "anytrust1",
}
```

And then keys and `local.data-dir` would be stored under `~/.arbitrum/anytrust1/` by default unless specified as absolute.

Reply:
Moved data-dir to localConfig.
Created https://github.com/OffchainLabs/nitro/issues/637 for persistent config.